CheckValues/Results.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Text;
+using WaterLog_Backend;
+using WaterLog_Backend.Models;
+using WaterLog_Backend.Controllers;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CheckValues
+{
+    public class Results
+    {
+
+        public MonitorsController getController()
+        {
+            var builder = new ConfigurationBuilder();
+            builder.AddUserSecrets<Startup>();
+            var config = builder.Build();
+            string mySecret = config["Localhost:ConnectionString"];
+
+            var optionsBuilder = new DbContextOptionsBuilder<DatabaseContext>();
+            optionsBuilder.UseSqlServer("Server=dev.retrotest.co.za;Database=iot;User Id=group1;Password=fNX^r+UKy3@CtYh5");
```

Comment:
This should really not be in the repo, especially since its public. You'll need to figure out how to get these values from the system properties and then have you details changed. 

You should assume this user and password has been compromised and is general knowledge of the world